### PR TITLE
Add Arabic translation

### PR DESCRIPTION
**Pull request by akarimg**

### CODE REVIEW

**Comment by brunonova** [comment]
Hi, thank you for your translation!

Could you also translate the file `polkit/org.freedesktop.nautilus-admin.policy`?

And two things about `po/ar.po`
- 3rd line: `Waqf Public license` doesn't seem right.
- 2nd line: the copyright line should have my name and e-mail, and then you should add after the 3rd line a new line with your `Name <email>, year.` (that's how it's usually done, from what I've seen)

Basically, the first lines should be something like this:

```
# Arabic translations for nautilus-admin package.
# Copyright (C) 2015 Bruno Nova <brunomb.nova@gmail.com>
# This file is distributed under the same license as the nautilus-admin package.
# Abdelkarim Guettaf <gkraim@gmail.com>, 2015.
#
```

**Comment by akarimg** [comment]
Hi, Thanks brunonova.
I have sent a new request with correction.
